starred.py

```diff
@@ -3,7 +3,7 @@
 import sys
 from collections import OrderedDict
 import click
-from github3 import GitHub
+from github3 import GitHub, login
```

Comment:
It seems that ‘import login’ is not necessary. https://github3py.readthedocs.io/en/master/github.html#github3.github.GitHub